src/installer/pkg/sfx/installers/host.wxs

```diff
@@ -20,19 +20,25 @@
         1. the legacy SDK can be subsequently installed
         2. the user runs 'dotnet' commands directly against 'ProgramFiles'\dotnet\dotnet.exe -->
 
-      <Component Id="cmpCoreHost" Guid="{45399BBB-DDA5-4386-A2E9-618FB3C54A18}" >
+      <Component Id="cmpCoreHost" Directory="DOTNETHOME" Guid="{45399BBB-DDA5-4386-A2E9-618FB3C54A18}" >
         <File Id="fileCoreHostExe" KeyPath="yes" Source="$(var.HostSrc)\dotnet.exe">
           <CopyFile Id="copyFileCoreHostExe" DestinationDirectory="PROGRAMFILES_DOTNET" />
         </File>
       </Component>
         
-      <Component Id="cmpSharedHostVersionRegistry" Guid="*">
+      <Component Id="cmpSharedHostVersionRegistry" Directory="DOTNETHOME" Guid="*">
```

Comment:
What does it mean to add Directory attributes to components that modify the registry?

Reply:
Every component must have a directory. Before the directory was applied at componentgroup level (I moved it there in a previous commit).  The directory is meaningless if the component doesn’t install any files.
https://docs.microsoft.com/en-us/windows/win32/msi/component-table